docs/source/refs.bib

```diff
@@ -88,6 +88,16 @@ @article{Cafaro_2014_PRA
   url       = {https://link.aps.org/doi/10.1103/PhysRevA.89.022316},
 }
 
+@article{Calderon_2023_Quantum,
+  title       = {Quantum circuit debugging and sensitivity analysis via local inversions},
+  author      = {Calderon-Vargas, Fernando A and Proctor, Timothy and Rudinger, Kenneth and Sarovar, Mohan},
+  journal     = {Quantum},
+  volume      = {7},
+  pages       = {921},
+  year        = {2023},
+  publisher   = {Verein zur F{\"o}rderung des Open Access Publizierens in den Quantenwissenschaften},
+}
```

Comment:
Adding a URL (perhaps https://doi.org/10.22331/q-2023-02-09-921) would be helpful here to make the bibliography more useful.

Reply:
Done!